frontend/src/Pages/landing-page/components/BottomHalfH/Section_2.tsx

```diff
@@ -0,0 +1,151 @@
+/*Designed by Sandra Ashipala https://github.com/sandramsc*/
+import HeaderText from '../shared/HeaderText'
+import { useForm } from 'react-hook-form'
+import {
+  MapPinIcon,
+  DevicePhoneMobileIcon,
+  EnvelopeIcon,
+} from '@heroicons/react/24/solid'
+
+function Section_2() {
```

Comment:
Can we divide this component into maybe two components? also the name of the component, we can make it something related to it content, this will make it easier to track the code,